.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+# GitHub Dependabot configuration file. References:
+# https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: /
+    groups:
+      github-actions:
+        patterns:
+          - "*" # Group all Actions updates into a single larger pull request
+    schedule:
+      interval: monthly
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: monthly
```

Comment:
While I've kept the update schedule as "monthly" here, I would suggest that we keep it to a "weekly" basis, as I'll be much faster than Dependabot and will render it useless :P

I think my use case here is about PRs serving as reminders, rather than as updates. But, I can also configure these PRs to get auto-merged if CI passes.

Reply:
I think monthly would be OK. 
In my experience, dependabot generates a lot of unwanted notifications, and they are usually not that security critical... in our case we're running a static site, so there are no big security concerns.

As for auto-merge, better not. It makes more sense to auto-merge then check if the page is loading as expected.

Reply:
> I think monthly would be OK.
In my experience, dependabot generates a lot of unwanted notifications, and they are usually not that security critical... in our case we're running a static site, so there are no big security concerns.

Yes, I agree. At the current release frequency for JupyterLite and jupyterlite-pyodide-kernel releases, we should be fine with monthly updates.

> As for auto-merge, better not. It makes more sense to auto-merge then check if the page is loading as expected.

I agree with this as well.